src/test/java/teammates/test/cases/browsertests/AdminEmailPageUiTest.java

```diff
@@ -175,15 +181,15 @@ private boolean isEmailComposeElementsPresent() {
     }
 
     private boolean hasStatusMessageRecipientEmailFormatError(String recipientName) throws Exception {
-        return emailPage.getStatus().contains(
+        return emailPage.getTextsForAllStatusMessagesToUser().contains(
```

Comment:
@LiHaoTan Sorry but, I have some doubts regarding this. `getPopulatedErrorMessage` returns a `String`, and isn't `String` itself an Object? And my IDE is showing `List#contains(Object)` only? <br>
![screenshot 445](https://user-images.githubusercontent.com/10681755/37402233-abe9f806-2782-11e8-9f06-ca981c2a4b85.png)

Reply:
Is this what you are asking?

```java
return emailPage.getTextsForAllStatusMessagesToUser().contains(
        (Object) getPopulatedErrorMessage(
                FieldValidator.EMAIL_ERROR_MESSAGE, recipientName,
                FieldValidator.EMAIL_FIELD_NAME, FieldValidator.REASON_INCORRECT_FORMAT,
                FieldValidator.EMAIL_MAX_LENGTH));

```

Reply:
@LiHaoTan Your thoughts regarding this.

Reply:
No, I'm not saying you should change anything. I'm assuming that you intentionally altered the semantics of the test because we now actually can test multiple status messages.

> `getPopulatedErrorMessage` returns a `String`, and isn't `String` itself an Object? And my IDE is showing `List#contains(Object)` only?

You are not wrong, every class extends an `Object`. Your IDE is only showing `List#contains(Object)` because of this:

Reply:
>Assuming the semantic change from `String#contains(CharSequence)` to `List#contains(Object)` is intended.

My apologies. I totally misinterpreted your comment and thought that there is some misunderstanding regarding the `.contains()`, and was confused because of that.

>I'm assuming that you intentionally altered the semantics of the test because we now actually can test multiple status messages.

Yes, This was intended.